app/dashboards/user_dashboard.rb

```diff
@@ -46,15 +50,14 @@ class UserDashboard < Administrate::BaseDashboard
   # on the model's form (`new` and `edit`) pages.
   FORM_ATTRIBUTES = %i[
     email
+    matches
+    receivers
```

Comment:
We don't need to add these two lines here, it would only be confusing because essentially it means we have an input in the admin user form to add matches and receivers. It's better if these are hidden. If we need to manually change something, it can be through the database console as to avoid some of us screweing up inadvertently.

Reply:
Got it